### ISSUE TITLE
Display exlcuded geofences on MADMin map when viewing an area

### ISSUE THREAD

**Issue by sn0opy** [Map-A-Droid/MAD#832]
Right now, the exluded geofences don't seem to be visible on the MADmin map. Neither as a separate geofence, nor as some sort of cutouts of the "included" geofences they're in.

**Comment by JabLuszko** [comment]
I would like to leave a note here that madmin it's not even showing geofences, it's showing areas using those geofences. No idea if that was on purpose, no idea if this issue shouldn't be blocked by some major geofence rework first.

**Comment by sn0opy** [comment]
That's correct. Being able to modify single geofences was something that came long after. Guess it would be helpfulp to display, add and modify geofences on the map too. 

But I guess that's something we should do after another sidebar rework. The current way makes things a bit complicated I think.
